app/views/claim/view-claim.html

```diff
@@ -71,30 +71,51 @@ <h1 class="table-heading">Visitor</h1>
   	  	<td>Prisoner Relationship</td>
         <td>{{ prisonerRelationshipsEnum[Claim['Relationship']].displayName }}</td>
   	  </tr>
+
+      <tr>
+        <td>Benefit Information</td>
+        <td>
+          {{ benefitsEnum[Claim['Benefit']].displayName }} <br/>
+          {% if Claim['DWPBenefitCheckerResult'] == 'YES' %}
+          <span class="text-success">DWP - Eligible</span>
+          {% elif Claim['DWPBenefitCheckerResult'] == 'NO' %}
+          <span class="text-warning">DWP - Not eligible</span>
+          {% else %}
+          <span class="text-warning">DWP - No information</span>
+          {% endif %}
+
+          <select class="form-control action-select" id="dwp-check" name="dwpCheck">
+            <option value="">Select</option>
+            <option {% if Claim['DWPCheck'] == 'APPROVED' %} selected {% endif %} value="APPROVED">Approve</option>
+            <option {% if Claim['DWPCheck'] == 'REQUEST-INFORMATION' %} selected {% endif %} value="REQUEST-INFORMATION">Request information</option>
+            <option {% if Claim['DWPCheck'] == 'REJECTED' %} selected {% endif %} value="REJECTED">Reject</option>
```

Comment:
Selection will be lost if validation errors occur elsewhere on the page.

We can either add another check here for `claimDecision['dwpCheck']` as sent in req.body, or persist decision even if validation errors occur, as we do with Nomis Check.

So, [line 45 of view-claim.js](https://github.com/ministryofjustice/apvs-internal-web/pull/43/files#diff-b691608c328da9d58ea7dbf6a5a227a3R45) would become:

```
data.claim.NomisCheck = req.body.nomisCheck
data.claim.DWPCheck = req.body.dwpCheck
```